src/geom/cell_hex20.C

```diff
@@ -184,23 +184,7 @@ Order Hex20::default_order() const
 std::unique_ptr<Elem> Hex20::build_side_ptr (const unsigned int i,
                                              bool proxy )
 {
-  libmesh_assert_less (i, this->n_sides());
-
-  if (proxy)
-    return libmesh_make_unique<Side<Quad8,Hex20>>(this,i);
-
-  else
-    {
-      std::unique_ptr<Elem> face = libmesh_make_unique<Quad8>();
```

Comment:
So, we've only been setting parent() on proxy sides, not on non-proxy sides.  I guess that's why my backwards compatibility commit was overkill?  I still can't replicate the MOOSE test failures, they keep passing for me, but I can certainly try being more backwards compatible in my backwards compatibility.